app/assets/stylesheets/application.scss

```diff
@@ -4,10 +4,92 @@
   body{
     text-align: center;
   }
+  .body_color{
+    background-color: darkkhaki;
+  }
```

Comment:
use empty line between each block { }
so it will be:
.div {
 ...
}

.span {}